patch/00.initial.patch

```diff
@@ -19,11 +19,11 @@ index 167866b..72eac49 100644
  {{ include "kube-prometheus-stack.labels" . | indent 4 }}
  spec:
  {{- if .Values.alertmanager.alertmanagerSpec.image }}
--  baseImage: {{ .Values.alertmanager.alertmanagerSpec.image.repository }}
+-  image: {{ .Values.alertmanager.alertmanagerSpec.image.repository }}:{{ .Values.alertmanager.alertmanagerSpec.image.tag }}
 +{{- if (.Values.Installation) }}
-+  baseImage: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.alertmanager.alertmanagerSpec.image.name }}"
++  image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.alertmanager.alertmanagerSpec.image.tag }}"
 +{{ else }}
-+  baseImage: "{{ .Values.image.registry }}/{{ .Values.alertmanager.alertmanagerSpec.image.name }}"
++  image: {{ .Values.alertmanager.alertmanagerSpec.image.repository }}:{{ .Values.alertmanager.alertmanagerSpec.image.tag }}
```

Comment:
Issue explained above. Registry missing. Would probably try to pull from docker hub